test/single_test.dart

```diff
@@ -100,4 +101,18 @@ END
     expect(e.message,
         'Length of parameters (1) does not match parameter count in query (2)');
   });
+
+  test('json type test', () async {
+    await conn.query('DROP TABLE IF EXISTS tjson');
+    await conn.query('CREATE TABLE tjson(a int, b json NULL)');
+    MySqlClientError e;
+    try {
+      await conn.query(
+          'INSERT INTO `tjson` (a, b) VALUES (?, ?)', [3, '{"test":"test"}']);
+      var result = await conn.query('SELECT * FROM tjson');
+      print(result.first.fields);
+    } on MySqlClientError catch (e1) {
```

Comment:
What is the purpose of the try-catch?

Reply:
Thank your reply，No purpose ， this is my copy code

Reply:
What do I need to modify

Reply:
What do I need to modify